WordPress/Tests/Theme/NoAutoGenerateUnitTest.inc

```diff
@@ -0,0 +1,11 @@
+
+<p>Lubith</p>
+
+<?php
+// Lubith
+echo "Lubith";
+// Templatetoaster
+// wpthemegenerator.com
+/*
+ * @package templatetoaster
+ */
```

Comment:
As you are sniffing for both PHP as well as CSS, you will need a css example file too and deal with that in the unit test file.